tests/e2e/tests/ndb_test.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from google.appengine.ext.ndb import model, tasklets
+
+class Employee(model.Model):
+  name = model.StringProperty()
+  age = model.IntegerProperty()
+
+
+@pytest.fixture
+def setup():
```

Comment:
call this `ctx` instead, and might as well return the `ctx` in case you want to use it in a test. :)